Kata.Test/CalculatorShould.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.VisualStudio.TestPlatform.Utilities;
+
+namespace Kata.Test;
+
+public class CalculatorShould
+{
+    [Theory]
+    [InlineData("1,2,3", 6)]
+    [InlineData("1,2", 3)]
+    [InlineData("1", 1)]
+    [InlineData("", 0)]
+    public void Add_More_Than_One_Value(string numbers, int output)
```

Comment:
use test name format serviceName_ShouldResult_WhenCase
in this case it will be: 
Add_ShouldReturnCorrectResult_WhenInputContainsMoreThanOneValue